cpp_utils/BLEUtils.cpp

```diff
@@ -1302,8 +1301,7 @@ void BLEUtils::dumpGattClientEvent(
 		// - uint16_t          conn_id
 		// - esp_bd_addr_t     remote_bda
 		case ESP_GATTC_DISCONNECT_EVT: {
-			ESP_LOGD(LOG_TAG, "[staus: %s, conn_id: %d, remote_bda: %s]",
-				BLEUtils::gattStatusToString(evtParam->disconnect.status).c_str(),
```

Comment:
Why remove the logging of status?

Reply:
esp-idf removed that field in commit 79f206be47c3f608615c1de8c491107e6c9194bb.